hack/e2e-node-test.sh

```diff
@@ -15,6 +15,11 @@
 # limitations under the License.
 
 # This script is a vestigial redirection.  Please do not add "real" logic.
+# The "true" target of this makerule is `hack/make-rules/test-e2e-node.sh`.
+
+# This script runs `make test-e2e-node` command.
+# The command builds and runs node end-to-end tests.
+# Usage: `hack/e2e-node-test.sh`.
```

Comment:
I would call out that this uses the FOCUS and SKIP env vars and provide an additional example